frontend/pages/hosts/details/HostDetailsPage/modals/UnenrollMdmModal/UnenrollMdmModal.tsx

```diff
@@ -52,43 +51,63 @@ const UnenrollMdmModal = ({
     onClose();
   };
 
+  const generateDescription = () => {
+    if (isIPadOrIPhone(hostPlatform)) {
+      return (
+        <>
+          <p>Settings configured by Fleet will be removed.</p>
+          <p>
+            To re-enroll, go to <b>Hosts &gt; Add hosts &gt; iOS/iPadOS</b> and
+            share the link with end user.
```

Comment:
Is the copy not wrong here? 

<img width="603" height="105" alt="Image" src="https://github.com/user-attachments/assets/e6553135-41b2-4122-ae62-e04d47691fdb" />